python/lsst/ts/wep/task/calcZernikesNeuralTask.py

```diff
@@ -0,0 +1,1964 @@
+        Model Weights Path for aggregatornet. If None, TARTS will create a new
+        model with random weights (useful for testing).
+    datasetParamPath : str
+        Path to TARTS dataset parameters YAML file containing normalization
+        scaling factors, image processing parameters (CROP_SIZE, deg_per_pix,
+        mm_pix), model hyperparameters, and training data file paths
+    device : str
+        Device to use for calculations
+    nollIndices : list[int]
+        List of Noll indices to calculate. Default is Z4-Z23 (4-23),
+        excluding piston (Z1), tip (Z2), and tilt (Z3) which are
+        typically not measured in wavefront sensing.
+    cropSize : int
+        Size of donut crop in pixels (width and height). Default is 160 pixels,
+        which matches the TARTS neural network training data format.
+    """
+
+    wavenetPath: pexConfig.Field = pexConfig.Field(
+        doc="Model Weights Path for wavenet",
+        dtype=str,
+        default=None,
+        optional=True
+    )
+    alignetPath: pexConfig.Field = pexConfig.Field(
+        doc="Model Weights Path for alignet",
+        dtype=str,
+        default=None,
+        optional=True
+    )
+    aggregatornetPath: pexConfig.Field = pexConfig.Field(
+        doc="Model Weights Path for aggregatornet",
+        dtype=str,
+        default=None,
+        optional=True
+    )
+    datasetParamPath: pexConfig.Field = pexConfig.Field(
+        doc="Path to TARTS dataset parameters YAML file containing normalization "
+            "scaling factors, image processing parameters (CROP_SIZE, deg_per_pix, "
+            "mm_pix), model hyperparameters, and training data file paths",
+        dtype=str
+    )
+    device: pexConfig.Field = pexConfig.Field(
+        doc="Device to use for calculations",
+        dtype=str,
+        default="cuda"
+    )
+    nollIndices: pexConfig.ListField = pexConfig.ListField(
+        doc="List of Noll indices to calculate. Default is Z4-Z23 (4-23), "
+            "excluding piston (Z1), tip (Z2), and tilt (Z3) which are "
+            "typically not measured in wavefront sensing.",
+        dtype=int,
+        default=[4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23]
+    )
+    cropSize: pexConfig.Field = pexConfig.Field(
+        doc="Size of donut crop in pixels (width and height). Default is 160 pixels, "
+            "which matches the TARTS neural network training data format.",
+        dtype=int,
+        default=160
+    )
+
+
+class CalcZernikesNeuralTask(pipeBase.PipelineTask):
+    """Neural network-based Zernike estimation task using TARTS.
+
+    This class uses the TARTS (Telescope Active Optics Real-Time System)
+    neural network models to estimate Zernike coefficients from pairs of
+    intra and extra-focal exposures. Each exposure contains donut stamps
+    from one focal position, and the task processes them separately to
+    estimate Zernike coefficients for each side of the focal plane.
```

Comment:
Maybe add two more sentences, detailing the three different networks and what they do? Just so the reader knows exactly what this task will be doing.